packages/react/src/components/Authenticator/ConfirmSignIn/ConfirmSignIn.tsx

```diff
@@ -37,8 +38,7 @@ export const ConfirmSignIn = (): JSX.Element => {
       onSubmit={handleSubmit}
     >
       <fieldset
-        style={{ display: 'flex', flexDirection: 'column' }}
-        className="amplify-flex"
+        className={classNames('amplify-flex', 'amplify-authenticator__column')}
```

Comment:
How about using `<Flex as="fieldset" direction="column" />` without the need of adding CSS class?